packages/bootstrap/scss/spreadsheet/_theme.scss

```diff
@@ -157,3 +167,23 @@
     }
 
 }
+
+
+
+
+@include exports("spreadsheet/rtl/theme") {
+
+    .k-rtl .k-spreadsheet,
+    .k-spreadsheet[dir="rtl"] {
+
+        .k-dirty {
+            border-color: $error $error transparent transparent;
```

Comment:
The same as 
https://github.com/telerik/kendo/pull/9584#discussion_r277264174

(there are similar sections below)